lib/ssl/src/ssl.erl

```diff
@@ -445,7 +445,6 @@
                                 {alpn_preferred_protocols, server_alpn()} |
                                 {next_protocols_advertised, server_next_protocol()} |
                                 {psk_identity, server_psk_identity()} |
-                                {honor_cipher_order, boolean()} |
```

Comment:
I was unsure this change alone would remake the doc. which is why I didn't do it myself 😄 Didn't want to go after doc. generation issues, as I have in the past 😄

Reply:
Sympathize with that ;)